solutions_python_book/lazy_student/lazy_student.py

```diff
@@ -0,0 +1,85 @@
+import sys
+
+
+def main():
+    file_home_work_name = "home_work.txt"
```

Comment:
constant values should be UPPER_CASE, 
and at the top of the file, after the imports